src/util/hass-util.html

```diff
@@ -231,16 +231,31 @@
   return 'display';
 };
 
+window.hassUtil.ensureCustomUILoaded = function (customUI) {
+  Polymer.Base.importHref(
+      '/local/custom_ui/' + customUI + '.html',
+      function () {},
+      /* eslint-disable no-console */
+      function () { console.error('Error loading %s from /local/custom_ui/%s.html', customUI, customUI); },
+      /* eslint-enable no-console */
+      true);
+};
+
 window.hassUtil.stateMoreInfoType = function (stateObj) {
+  if (stateObj.attributes && 'custom_ui_more_info' in stateObj.attributes) {
+    var customUI = stateObj.attributes.custom_ui_more_info;
+    window.hassUtil.ensureCustomUILoaded(customUI);
+    return customUI;
+  }
   var domain = window.hassUtil.computeDomain(stateObj);
 
   if (window.hassUtil.DOMAINS_WITH_MORE_INFO.indexOf(domain) !== -1) {
-    return domain;
+    return 'MORE-INFO-' + domain;
```

Comment:
Since the return value of this function need to be uppercased anyway, I think it is more readable to use lower-case here.